test/Confluent.Kafka.UnitTests/Confluent.Kafka.UnitTests.csproj

```diff
@@ -16,4 +16,8 @@
     <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Visual studio keeps adding it, I think we could include it. Didn't look if they changed it in 2.0